test/unit/sso/openid_connect_test.rb

```diff
@@ -86,6 +86,24 @@ class OpenidConnectTest < ActiveSupport::TestCase
     end
   end
 
+  describe "logout_url" do
+    let(:subject) do
+      token = JWT.encode(payload, nil, 'none')
+      SSO::OpenidConnect.new api_controller({:authorization => "Bearer #{token}"})
+    end
+
+    test "it returns correct external logout path" do
+      SSO::OpenidConnect.any_instance.stubs(:mod_auth_openidc_session_exist?).returns(true)
+      logout_url = Setting[:login_delegation_logout_url] + "/redirect_uri?logout=" + CGI.escape(Setting[:login_delegation_logout_url])
```

Comment:
I'd explicitly set a value for this setting and then test it. Then you can also leave out the fixture. IMHO fixtures should be avoided and you should explicitly control the things you depend on in your tests.